<a href="https://colab.research.google.com/github/benoynoel/Fortnite2.github.io/blob/master/Hypothesis_Testing_Assignment(2_Sample_T_Test).ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
import pandas as pd
import numpy as np
from scipy import stats
df = pd.read_csv("Cutlets.csv")

In [ ]:
df

Unit A  Unit B
0   6.8090  6.7703
1   6.4376  7.5093
2   6.9157  6.7300
3   7.3012  6.7878
4   7.4488  7.1522
5   7.3871  6.8110
6   6.8755  7.2212
7   7.0621  6.6606
8   6.6840  7.2402
9   6.8236  7.0503
10  7.3930  6.8810
11  7.5169  7.4059
12  6.9246  6.7652
13  6.9256  6.0380
14  6.5797  7.1581
15  6.8394  7.0240
16  6.5970  6.6672
17  7.2705  7.4314
18  7.2828  7.3070
19  7.3495  6.7478
20  6.9438  6.8889
21  7.1560  7.4220
22  6.5341  6.5217
23  7.2854  7.1688
24  6.9952  6.7594
25  6.8568  6.9399
26  7.2163  7.0133
27  6.6801  6.9182
28  6.9431  6.3346
29  7.0852  7.5459
30  6.7794  7.0992
31  7.2783  7.1180
32  7.1561  6.6965
33  7.3943  6.5780
34  6.9405  7.3875

#Mean and Sd of Unit A

In [ ]:
[df['Unit A'].mean()],[df['Unit A'].std()]

([7.0190914285714285], [0.28840848418154963])

#Mean and Sd of Unit B

In [ ]:
[df['Unit B'].mean()],[df['Unit B'].std()]

([6.964297142857142], [0.343400647063108])

#T-critical value


In [ ]:
stats.t.ppf(0.975,34)

2.032244509317718

#P-Value

In [ ]:
Unit_A = df['Unit A']
Unit_B = df['Unit B']
stats.ttest_ind(Unit_A, Unit_B)

Ttest_indResult(statistic=0.7228688704678063, pvalue=0.4722394724599501)

P-value > Alpha value i.e.,
0.4722 > 0.05
There is no significant difference between two Units